Makefile

```diff
@@ -20,24 +21,29 @@ setup: setup-python setup-databases
 setup-python:
 	@read -p 'Did you create and activated a dedicated virtualenv? [y/N]: '; \
 	if [[ $$REPLY = y ]]; then \
-		pip install -r tests-requirements.txt; \
+		pip install --user -r tests-requirements.txt; \
 	else \
-		echo 'Aborting'; exit 1; \
+		echo 'Virtual env not installed. Installing for the local user...'; \
+		pip install --user virtualenv; \
+		virtualenv .; \
+		echo 'Virtual env installed and configured!'; \
+		echo 'Calling setup again, just wait and enjoy :)'; \
+		exit 0 && make setup; \
 	fi
```

Comment:
Thanks for the explanations, I think you can reject this pull request or just use parte of it in your code.